<!DOCTYPE html>
<html class="client-js"><head>
  <meta charset=UTF-8"UTF-8">
  <title>Pyruvate decarboxylase</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <script>
    function importScript() { return 1 } // this is to avoid the error from site.js
  </script>
  <link rel="canonical" href="https://en.wikipedia.org/wiki/Pyruvate_decarboxylase">
  <link href="../-/s/css_modules/ext.cite.ux-enhancements.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/ext.cite.ux-enhancements.css" rel="stylesheet" type="text/css" class="">
<link href="../-/s/css_modules/ext.cite.styles.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/ext.cite.styles.css" rel="stylesheet" type="text/css" class="">
<link href="../-/s/css_modules/mediawiki.page.gallery.styles.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/mediawiki.page.gallery.styles.css" rel="stylesheet" type="text/css" class="">
<link href="../-/s/css_modules/fi000001.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/skins.minerva.base.reset|skins.minerva.content.styles|ext.cite.style|site.styles|mobile.app.pagestyles.android|mediawiki.page.gallery.styles|mediawiki.skinning.content.parsoid.css" rel="stylesheet" type="text/css" class="">
  <link href="../-/s/css_modules/style.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/style.css" rel="stylesheet" type="text/css" class=""><link href="../-/s/css_modules/content.parsoid.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/content.parsoid.css" rel="stylesheet" type="text/css" class=""><link href="../-/s/css_modules/inserted_style.css" tppabs="https://ptable.com/wiki/compounds/-/s/css_modules/inserted_style.css" rel="stylesheet" type="text/css" class="">
  <script src="../-/j/js_modules/script.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/script.js" class=""></script><script src="../-/j/js_modules/masonry.min.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/masonry.min.js" class=""></script><script src="../-/j/js_modules/article_list_home.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/article_list_home.js" class=""></script><script src="../-/j/js_modules/images_loaded.min.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/images_loaded.min.js" class=""></script><script src="../-/j/js_modules/node_module/details-element-polyfill/dist/details-element-polyfill.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/node_module/details-element-polyfill/dist/details-element-polyfill.js" class=""></script>
<link type="root" href=""></head>

<body class="mediawiki mw-hide-empty-elt ns-0 ns-subject stable skin-minerva action-view animations">
  <div id="mw-mf-viewport" class="feature-header-v2">
    <div id="mw-mf-page-center">
      <div id="content" class="mw-body">
        <a id="top"></a>
        <div id="bodyContent" class="content mw-parser-output">
          
          <div id="mw-content-text" style="direction: ltr;"><h1 class="section-heading" tabindex="0" aria-haspopup="true" data-section-id="0">
  <span class="mw-headline" id="title_0">Pyruvate decarboxylase</span>
</h1>
<div id="mf-section-0" class="mf-section-0" aria-pressed="true" aria-expanded="true">
  <p><b>Pyruvate decarboxylase</b> is a homotetrameric enzyme (EC<span> </span><a href="javascript:if(confirm('https://enzyme.expasy.org/EC/4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://enzyme.expasy.org/EC/4.1.1.1'" tppabs="https://enzyme.expasy.org/EC/4.1.1.1" class="external text external">4.1.1.1</a>) that catalyses the decarboxylation of <a href="Pyruvic_acid.htm" tppabs="https://ptable.com/wiki/compounds/A/Pyruvic_acid" title="Pyruvic acid">pyruvic acid</a> to <a href="Acetaldehyde.htm" tppabs="https://ptable.com/wiki/compounds/A/Acetaldehyde" title="Acetaldehyde">acetaldehyde</a> and <a href="Carbon_dioxide.htm" tppabs="https://ptable.com/wiki/compounds/A/Carbon_dioxide" title="Carbon dioxide">carbon dioxide</a> in the cytoplasm of prokaryotes, and in the cytoplasm and mitochondria of eukaryotes. It is also called 2-oxo-acid carboxylase, alpha-ketoacid carboxylase, and pyruvic decarboxylase.<span class="mw-ref" id="cite_ref-urlENZYME_entry_4.1.1.1_1-0"><a href="#cite_note-urlENZYME_entry_4.1.1.1-1" style="counter-reset: mw-Ref 1;"><span class="mw-reflink-text">[1]</span></a></span>  In anaerobic conditions, this enzyme is part of the fermentation process that occurs in yeast, especially of the genus <i>Saccharomyces</i>, to produce <a href="Ethanol.htm" tppabs="https://ptable.com/wiki/compounds/A/Ethanol" title="Ethanol">ethanol</a> by fermentation. It is also present in some species of fish (including goldfish and carp) where it permits the fish to perform ethanol fermentation (along with lactic acid fermentation) when oxygen is scarce.<span class="mw-ref" id="cite_ref-2"><a href="#cite_note-2" style="counter-reset: mw-Ref 2;"><span class="mw-reflink-text">[2]</span></a></span> Pyruvate decarboxylase starts this process by converting pyruvate into acetaldehyde and carbon dioxide.<span class="mw-ref" id="cite_ref-isbn0-9747077-1-6_3-0"><a href="#cite_note-isbn0-9747077-1-6-3" style="counter-reset: mw-Ref 3;"><span class="mw-reflink-text">[3]</span></a></span>  Pyruvate decarboxylase depends on cofactors <a href="Thiamine_pyrophosphate.htm" tppabs="https://ptable.com/wiki/compounds/A/Thiamine_pyrophosphate" title="Thiamine pyrophosphate">thiamine pyrophosphate</a> (TPP) and magnesium. This enzyme should not be mistaken for the unrelated enzyme pyruvate dehydrogenase, an oxidoreductase (EC<span> </span><a href="javascript:if(confirm('https://enzyme.expasy.org/EC/1.2.4.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://enzyme.expasy.org/EC/1.2.4.1'" tppabs="https://enzyme.expasy.org/EC/1.2.4.1" class="external text external">1.2.4.1</a>), that catalyzes the oxidative decarboxylation of pyruvate to <a href="Acetyl-CoA.htm" tppabs="https://ptable.com/wiki/compounds/A/Acetyl-CoA" title="Acetyl-CoA">acetyl-CoA</a>.</p>


<table class="infobox" style="width:22em"><tbody><tr><th colspan="2" style="text-align:center;font-size:125%;font-weight:bold">Pyruvate decarboxylase</th></tr><tr><td colspan="2" style="text-align:center"><figure-inline class="mw-default-size"><img src="../I/m/Pyruvate_decarb_1.svg.png" tppabs="https://ptable.com/wiki/compounds/I/m/Pyruvate_decarb_1.svg.png" data-file-width="750" data-file-height="245" data-file-type="drawing" height="72" width="220"></figure-inline><div>Reaction catalyzed by pyruvate decarboxylase:<br>pyruvate + thiamine pyrophosphate (TPP) → hydroxyethyl-TPP + CO<sub>2</sub></div></td></tr><tr><th colspan="2" style="text-align:center;background-color: #ddd">Identifiers</th></tr><tr><th scope="row" style="background-color: #e7dcc3">EC number</th><td style="background-color: #eee"><a href="javascript:if(confirm('https://www.qmul.ac.uk/sbcs/iubmb/enzyme/EC4/1/1/1.html  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.qmul.ac.uk/sbcs/iubmb/enzyme/EC4/1/1/1.html'" tppabs="https://www.qmul.ac.uk/sbcs/iubmb/enzyme/EC4/1/1/1.html" class="external text external">4.1.1.1</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">CAS number</th><td style="background-color: #eee"><a href="javascript:if(confirm('http://www.commonchemistry.org/ChemicalDetail.aspx?ref=9001-04-1&title=  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://www.commonchemistry.org/ChemicalDetail.aspx?ref=9001-04-1&title='" tppabs="http://www.commonchemistry.org/ChemicalDetail.aspx?ref=9001-04-1&title=" class="external text external">9001-04-1 </a></td></tr><tr><th colspan="2" style="text-align:center;background-color: #ddd">Databases</th></tr><tr><th scope="row" style="background-color: #e7dcc3">IntEnz</th><td style="background-color: #eee"><a href="javascript:if(confirm('https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=4.1.1.1'" tppabs="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=4.1.1.1" class="external text external">IntEnz view</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">BRENDA</th><td style="background-color: #eee"><a href="javascript:if(confirm('http://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.1'" tppabs="http://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.1" class="external text external">BRENDA entry</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">ExPASy</th><td style="background-color: #eee"><a href="javascript:if(confirm('http://www.expasy.org/enzyme/4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://www.expasy.org/enzyme/4.1.1.1'" tppabs="http://www.expasy.org/enzyme/4.1.1.1" class="external text external">NiceZyme view</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">KEGG</th><td style="background-color: #eee"><a href="javascript:if(confirm('https://www.genome.jp/dbget-bin/www_bget?enzyme+4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.genome.jp/dbget-bin/www_bget?enzyme+4.1.1.1'" tppabs="https://www.genome.jp/dbget-bin/www_bget?enzyme+4.1.1.1" class="external text external">KEGG entry</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">MetaCyc</th><td style="background-color: #eee"><a href="javascript:if(confirm('https://biocyc.org/META/substring-search?type=NIL&object=4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://biocyc.org/META/substring-search?type=NIL&object=4.1.1.1'" tppabs="https://biocyc.org/META/substring-search?type=NIL&object=4.1.1.1" class="external text external">metabolic pathway</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">PRIAM</th><td style="background-color: #eee"><a href="javascript:if(confirm('http://priam.prabi.fr/cgi-bin/PRIAM_profiles_CurrentRelease.pl?EC=4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://priam.prabi.fr/cgi-bin/PRIAM_profiles_CurrentRelease.pl?EC=4.1.1.1'" tppabs="http://priam.prabi.fr/cgi-bin/PRIAM_profiles_CurrentRelease.pl?EC=4.1.1.1" class="external text external">profile</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">PDB structures</th><td style="background-color: #eee"><a href="javascript:if(confirm('http://www.rcsb.org/pdb/search/smartSubquery.do?smartSearchSubtype=EnzymeClassificationQuery&Enzyme_Classification=4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://www.rcsb.org/pdb/search/smartSubquery.do?smartSearchSubtype=EnzymeClassificationQuery&Enzyme_Classification=4.1.1.1'" tppabs="http://www.rcsb.org/pdb/search/smartSubquery.do?smartSearchSubtype=EnzymeClassificationQuery&Enzyme_Classification=4.1.1.1" class="external text external">RCSB PDB</a> <a href="javascript:if(confirm('https://www.ebi.ac.uk/pdbe/entry/search/index?ec_number:4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/pdbe/entry/search/index?ec_number:4.1.1.1'" tppabs="https://www.ebi.ac.uk/pdbe/entry/search/index?ec_number:4.1.1.1" class="external text external">PDBe</a> <a href="javascript:if(confirm('https://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=4.1.1.1'" tppabs="https://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/enzymes/GetPage.pl?ec_number=4.1.1.1" class="external text external">PDBsum</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3">Gene Ontology</th><td style="background-color: #eee"><a href="javascript:if(confirm('http://amigo.geneontology.org/amigo/term/GO:0004737  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://amigo.geneontology.org/amigo/term/GO:0004737'" tppabs="http://amigo.geneontology.org/amigo/term/GO:0004737" class="external text external">AmiGO </a> / <a href="javascript:if(confirm('https://www.ebi.ac.uk/QuickGO/term/GO:0004737  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/QuickGO/term/GO:0004737'" tppabs="https://www.ebi.ac.uk/QuickGO/term/GO:0004737" class="external text external">QuickGO</a></td></tr><tr><td colspan="2" style="text-align:center;background-color: #eee"><table class="infobox collapsible collapsed" style="width:22em;float:none; clear:none; margin:0; border-width:0; border-collapse:collapse; text-align:left; width:100%"><tbody><tr><th colspan="2" style="text-align:center;background-color: #ddd">Search</th></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PMC</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;"><a href="javascript:if(confirm('https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&term=4.1.1.1%5BEC/RN%20Number%5D%20AND%20pubmed%20pmc%20local%5Bsb%5D  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&term=4.1.1.1%5BEC/RN%20Number%5D%20AND%20pubmed%20pmc%20local%5Bsb%5D'" tppabs="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&term=4.1.1.1%5BEC/RN%20Number%5D%20AND%20pubmed%20pmc%20local%5Bsb%5D" class="external text external">articles</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">PubMed</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;"><a href="javascript:if(confirm('https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&term=4.1.1.1%5BEC/RN%20Number%5D  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&term=4.1.1.1%5BEC/RN%20Number%5D'" tppabs="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&term=4.1.1.1%5BEC/RN%20Number%5D" class="external text external">articles</a></td></tr><tr><th scope="row" style="background-color: #e7dcc3; border:#fafafa 2px solid; border-width:3px 2px 0 0;">NCBI</th><td style="background-color: #eee; border:#fafafa 2px solid; border-width:3px 0 0 2px;"><a href="javascript:if(confirm('https://www.ncbi.nlm.nih.gov/protein?term=4.1.1.1%5BEC/RN%20Number%5D  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.ncbi.nlm.nih.gov/protein?term=4.1.1.1%5BEC/RN%20Number%5D'" tppabs="https://www.ncbi.nlm.nih.gov/protein?term=4.1.1.1%5BEC/RN%20Number%5D" class="external text external">proteins</a></td></tr></tbody></table></td></tr></tbody></table>


</div><details data-level="2" open="">
    <summary class="section-heading"><h2 id="Structure">Structure</h2></summary>
    
<p>Pyruvate decarboxylase occurs as a dimer of dimers with two active sites shared between the monomers of each dimer. The enzyme contains a beta-alpha-beta structure, yielding parallel beta-sheets.  It contains 563 residue subunits in each dimer; the enzyme has strong intermonomer attractions, but the dimers loosely interact to form a loose tetramer.<span class="mw-ref" id="cite_ref-pmid8512926_4-0"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span></p>

<style data-mw-deduplicate="TemplateStyles:r959344415">@media all and (max-width:720px){.mw-parser-output .mod-gallery{width:100%!important}}.mw-parser-output .mod-gallery{display:table}.mw-parser-output .mod-gallery-default{background:transparent;margin-top:4px}.mw-parser-output .mod-gallery-center{margin-left:auto;margin-right:auto}.mw-parser-output .mod-gallery-left{float:left}.mw-parser-output .mod-gallery-right{float:right}.mw-parser-output .mod-gallery-none{float:none}.mw-parser-output .mod-gallery-collapsible{width:100%}.mw-parser-output .mod-gallery .title,.mw-parser-output .mod-gallery .main,.mw-parser-output .mod-gallery .footer{display:table-row}.mw-parser-output .mod-gallery .title>div{display:table-cell;text-align:center;font-weight:bold}.mw-parser-output .mod-gallery .main>div{display:table-cell}.mw-parser-output .mod-gallery .caption{display:table-row;vertical-align:top}.mw-parser-output .mod-gallery .caption>div{display:table-cell;display:block;font-size:94%;padding:0}.mw-parser-output .mod-gallery .footer>div{display:table-cell;text-align:right;font-size:80%;line-height:1em}.mw-parser-output .mod-gallery .gallerybox img{background:none!important}.mw-parser-output .mod-gallery .bordered-images .thumb img{border:solid #eee 1px}.mw-parser-output .mod-gallery .whitebg .gallerybox,.mw-parser-output .mod-gallery .whitebg .thumb{background:#fff!important}.mw-parser-output .mod-gallery .whitebg .thumb{margin-top:1px}</style><div class="mod-gallery mod-gallery-default"><div class="title"><div>Crystallographic structures of pyruvate decarboxylase</div></div><div class="main"><div><ul class="gallery mw-gallery-traditional nochecker bordered-images whitebg" style="line-height:130%;">
<li class="gallerybox" style="width: 310px;"><div class="thumb" style="width: 305px; height: 210px;"><figure-inline><img src="../I/m/1pvd_with_TPP.jpg" tppabs="https://ptable.com/wiki/compounds/I/m/1pvd_with_TPP.jpg" data-file-width="450" data-file-height="346" data-file-type="bitmap" height="180" width="234"></figure-inline></div><div class="gallerytext">Cartoon diagram of pyruvate decarboxylase monomer with TPP attached.</div></li>
<li class="gallerybox" style="width: 310px;"><div class="thumb" style="width: 305px; height: 210px;"><figure-inline><img src="../I/m/1pvd_image.jpg" tppabs="https://ptable.com/wiki/compounds/I/m/1pvd_image.jpg" data-file-width="250" data-file-height="250" data-file-type="bitmap" height="180" width="180"></figure-inline></div><div class="gallerytext">Cartoon diagram of pyruvate decarboxylase tetramer.</div></li>
<li class="gallerybox" style="width: 310px;"><div class="thumb" style="width: 305px; height: 210px;"><figure-inline><img src="../I/m/PDC_chimera_cropped_r3_c4.jpg" tppabs="https://ptable.com/wiki/compounds/I/m/PDC_chimera_cropped_r3_c4.jpg" data-file-width="312" data-file-height="317" data-file-type="bitmap" height="180" width="177"></figure-inline></div><div class="gallerytext">Active site of pyruvate decarboxylase with selected amino acids: Glu-51, Glu-477, Asp-444, and Asp-28.  Also displayed are cofactors TPP and Mg<sup>2+</sup>.</div></li>
<li class="gallerybox" style="width: 310px;"><div class="thumb" style="width: 305px; height: 210px;"><figure-inline><img src="../I/m/Hist_and_cyst_edited.jpg" tppabs="https://ptable.com/wiki/compounds/I/m/Hist_and_cyst_edited.jpg" data-file-width="592" data-file-height="615" data-file-type="bitmap" height="180" width="173"></figure-inline></div><div class="gallerytext">Positions of His and Cys residues in respect to active site (TPP and Mg) that participate in conformation changes when interacting with pyruvate substrate.</div></li>
</ul></div></div></div>


    <details data-level="3" open="">
    <summary class="section-heading"><h3 id="Active_site_residues">Active site residues</h3></summary>
    


<p>This enzyme is a homotetramer, and therefore has four active sites. The active sites are inside a cavity in the core of the enzyme where hydrogen bonding can occur and where the pyruvate reacts with TPP. Each active site has 20 amino acids, including the acidic Glu-477 (contributes to the stability of the TPP ring) and Glu-51 (aids in cofactor binding).  These Glutamates also contribute to forming the TPP ylid, acting as proton donators to the TPP aminopyrimidine ring.  The microenvironment around this Glu 477 is very nonpolar, contributing to a higher than normal pK<sub>a</sub> (normal Glu and Asp pKa's are around 4.6 in small proteins).<span class="mw-ref" id="cite_ref-Lobel_1996_5-0"><a href="#cite_note-Lobel_1996-5" style="counter-reset: mw-Ref 5;"><span class="mw-reflink-text">[5]</span></a></span></p>

<p>The lipophilic residues Ile-476, Ile-480 and Pro-26 contribute to the nonpolarity of the area around Glu-477. The only other negatively charged residue apart from TPP coenzyme is the Asp-28, which also aids in increasing the pK<sub>a</sub> of Glu-477.  Thus, the environment of the enzyme must allow for the protonation of the gamma-carboxyl group of Glu-477 to be around pH 6.<span class="mw-ref" id="cite_ref-Lobel_1996_5-1"><a href="#cite_note-Lobel_1996-5" style="counter-reset: mw-Ref 5;"><span class="mw-reflink-text">[5]</span></a></span></p>
 
<p>The aminopyrimidine ring on TPP acts as a base, once in its imine form, to pull off the C2 proton from TPP to form the nucleophile ylide.<span class="mw-ref" id="cite_ref-pmid8512926_4-1"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span> This must occur because the enzyme has no basic side chains present to deprotonate the TPP C2.  A mutation at the active site involving these Glu can result in the inefficiency or inactivity of the enzyme.  This inactivity has been proven in experiments in which either the N1' and/or 4'-amino groups are missing.  In NMR analysis, it has been determined that when TPP is bound to the enzyme along with the substate-analog pyruvamide, the rate of ylid formation is greater than the normal enzyme rate.  Also, the rate of mutation of Glu 51 to Gln reduces this rate significantly.<span class="mw-ref" id="cite_ref-pmid8512926_4-2"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span></p>

<p>Also included are Asp-444 and Asp-28 which stabilize the active site. These act as stabilizers for the Mg<sup>2+</sup> ion that is present in each active site. To ensure that only pyruvate binds, two Cys-221 (more than 20 Ångstroms away from each site) and His-92 trigger a conformational change which inhibits or activates the enzyme depending on the substrate that interacts with it.  If the substrate bound in the active site is pyruvate, then the enzyme is activated by a conformational change in this regulatory site.<span class="mw-ref" id="cite_ref-pmid9477950_6-0"><a href="#cite_note-pmid9477950-6" style="counter-reset: mw-Ref 6;"><span class="mw-reflink-text">[6]</span></a></span> The conformational change involves a 1,2 nucleophilic addition. This reaction, the formation of a thioketal, transforms the enzyme from its inactive to active state.</p>

<p>Inhibition of the site is done by a XC<sub>6</sub>H<sub>4</sub>CH=CHCOCOOH class of inhibitors/substrate analogues, as well as by the product of decarboxylation from such compounds as cinnamaldehydes. Other potential nucleophilic sites for the inhibitor include Cys-152, Asp-28, His-114, His-115, and Gln-477.<span class="mw-ref" id="cite_ref-pmid9477950_6-1"><a href="#cite_note-pmid9477950-6" style="counter-reset: mw-Ref 6;"><span class="mw-reflink-text">[6]</span></a></span></p>

<p>The normal catalytic rate of pyruvate decarboxylase is k<sub>cat</sub> = 10 s<sup>−1</sup>.  However, the rate of the enzyme with a Glu-51 mutation to Gln is 1.7 s<sup>−1</sup>.<span class="mw-ref" id="cite_ref-pmid8512926_4-3"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span></p>


</details>
<details data-level="3" open="">
    <summary class="section-heading"><h3 id="TPP_prosthetic_group">TPP prosthetic group</h3></summary>
    

<p>The cofactor TPP, C<sub>12</sub> H<sub>18</sub> N<sub>4</sub> O<sub>7</sub> P<sub>2</sub> S, is needed for this reaction's mechanism; it acts as the prosthetic group to the enzyme. The carbon atom between the sulfur and nitrogen atoms on thiazole ring act as carbanion which binds to the pyruvate.
TPP has an acidic H<sup>+</sup> on its C2 that acts as the functional part of the thiazolium ring; the ring acts as an "electron sink", enabling the carbanion electrons to be stabilized by resonance.<span class="mw-ref" id="cite_ref-isbn0-9747077-1-6_3-1"><a href="#cite_note-isbn0-9747077-1-6-3" style="counter-reset: mw-Ref 3;"><span class="mw-reflink-text">[3]</span></a></span>  The TPP can then act as a nucleophile with the loss of this C2 hydrogen, forming the ylide form of TPP.  This ylide can then attack pyruvate, which is held by the enzyme pyruvate decarboxylase.  During the decarboxylation of pyruvate, the TPP stabilizes the carbanion intermediates as an electrophile by noncovalent bonds.<span class="mw-ref" id="cite_ref-pmid8512926_4-4"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span>  Specifically, the pyridyl nitrogen N1' and the 4'-amino group of TPP are essential for the catalytic function of the enzyme-TDP complex.<span class="mw-ref" id="cite_ref-Lobel_1996_5-2"><a href="#cite_note-Lobel_1996-5" style="counter-reset: mw-Ref 5;"><span class="mw-reflink-text">[5]</span></a></span></p>


</details>

</details><details data-level="2" open="">
    <summary class="section-heading"><h2 id="Mechanism">Mechanism</h2></summary>
    <p>

The enzyme splits pyruvate into carbon dioxide and a 2-carbon fragment which is attached to its cofactor TPP.  This 2-carbon fragment is attached to the five membered TPP ring in its ylide form. This is an irreversible step in which it creates the acetaldehyde for the second step of alcoholic fermentation (anaerobic fermentation), in which acetaldehyde is reduced by NADH with alcohol dehydrogenase to ethanol.<span class="mw-ref" id="cite_ref-7"><a href="#cite_note-7" style="counter-reset: mw-Ref 7;"><span class="mw-reflink-text">[7]</span></a></span></p><div class="thumb tright"><div class="thumbinner" style="width:222px"><img src="../I/m/Pyruvate_decarboxylase_mechanism.png" tppabs="https://ptable.com/wiki/compounds/I/m/Pyruvate_decarboxylase_mechanism.png" data-file-width="627" data-file-height="157" data-file-type="bitmap" height="55" width="220"><div class="thumbcaption" style="text-align: left">Mechanism of pyruvate decarboxylation</div></div></div>


    
</details><details data-level="2" open="">
    <summary class="section-heading"><h2 id="Yeast">Yeast</h2></summary>
    

<p>In yeast, pyruvate decarboxylase acts independently during <a href="javascript:if(confirm('https://en.wiktionary.org/wiki/anaerobic  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://en.wiktionary.org/wiki/anaerobic'" tppabs="https://en.wiktionary.org/wiki/anaerobic" title="wikt:anaerobic" class="external">anaerobic</a> fermentation and releases the 2-carbon fragment as acetaldehyde plus carbon dioxide. Pyruvate decarboxylase creates the means of CO<sub>2</sub> elimination, which the cell dispels. The enzyme is also means to create ethanol, which is used as an antibiotic to eliminate competing organisms.<span class="mw-ref" id="cite_ref-pmid8512926_4-5"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span>  The enzyme is necessary to help the decarboxylation of alpha-keto acids because there is a build-up of negative charge that occurs on the carbonyl carbon atom in the transition state; therefore, the enzyme provides the suitable environment for TPP and the alpha-keto acid (pyruvate) to meet.<span class="mw-ref" id="cite_ref-pmid8512926_4-6"><a href="#cite_note-pmid8512926-4" style="counter-reset: mw-Ref 4;"><span class="mw-reflink-text">[4]</span></a></span></p>


    
</details><details data-level="2" open="">
    <summary class="section-heading"><h2 id="References">References</h2></summary>
    
<div class="reflist columns references-column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em; list-style-type: decimal;">
<ol class="mw-references references"><li id="cite_note-urlENZYME_entry_4.1.1.1-1"> <span id="mw-reference-text-cite_note-urlENZYME_entry_4.1.1.1-1" class="mw-reference-text"><cite class="citation web cs1"><a href="javascript:if(confirm('http://us.expasy.org/enzyme/4.1.1.1  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='http://us.expasy.org/enzyme/4.1.1.1'" tppabs="http://us.expasy.org/enzyme/4.1.1.1" class="external text external">"NiceZyme View of ENZYME: EC 4.1.1.1"</a>. ExPASy Proteomics Server.</cite></span></li><li id="cite_note-2"> <span id="mw-reference-text-cite_note-2" class="mw-reference-text"><cite id="CITEREFAren_van_WaardeG._Van_den_ThillartMaria_Verhagen1993" class="citation book cs1"><span class="new">Aren van Waarde</span>; G. Van den Thillart; <span class="new">Maria Verhagen</span> (1993). "Ethanol Formation and pH-Regulation in Fish". <i>Surviving Hypoxia</i>. pp.<span>&nbsp;</span>157−170. hdl:<a href="javascript:if(confirm('https://hdl.handle.net/11370%2F3196a88e-a978-4293-8f6f-cd6876d8c428  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://hdl.handle.net/11370%2F3196a88e-a978-4293-8f6f-cd6876d8c428'" tppabs="https://hdl.handle.net/11370%2F3196a88e-a978-4293-8f6f-cd6876d8c428" class="external text external">11370/3196a88e-a978-4293-8f6f-cd6876d8c428</a>. ISBN<span>&nbsp;</span><bdi>0-8493-4226-0</bdi>.</cite></span></li><li id="cite_note-isbn0-9747077-1-6-3"> <span id="mw-reference-text-cite_note-isbn0-9747077-1-6-3" class="mw-reference-text"><cite id="CITEREFTadhg_P._BegleyMcMurry,_John2005" class="citation book cs1">Tadhg P. Begley; McMurry, John (2005). <i>The organic chemistry of biological pathways</i>. Roberts and Co. Publishers. p.<span>&nbsp;</span>179. ISBN<span>&nbsp;</span><bdi>0-9747077-1-6</bdi>.</cite></span></li><li id="cite_note-pmid8512926-4"> <span id="mw-reference-text-cite_note-pmid8512926-4" class="mw-reference-text"><span class="">PDB: <a href="javascript:if(confirm('https://www.rcsb.org/structure/1pyd  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.rcsb.org/structure/1pyd'" tppabs="https://www.rcsb.org/structure/1pyd" class="external text external">1pyd</a></span><span>​</span>; <cite id="CITEREFDydaFureySwaminathanSax1993" class="citation journal cs1">Dyda F, Furey W, Swaminathan S, Sax M, Farrenkopf B, Jordan F (June 1993). "Catalytic centers in the thiamin diphosphate dependent enzyme pyruvate decarboxylase at 2.4-A resolution". <i>Biochemistry</i>. <b>32</b> (24): 6165–70. doi:<a href="javascript:if(confirm('https://doi.org/10.1021%2Fbi00075a008  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://doi.org/10.1021%2Fbi00075a008'" tppabs="https://doi.org/10.1021%2Fbi00075a008" class="external text external">10.1021/bi00075a008</a>. PMID<span>&nbsp;</span><a href="javascript:if(confirm('https://pubmed.ncbi.nlm.nih.gov/8512926  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://pubmed.ncbi.nlm.nih.gov/8512926'" tppabs="https://pubmed.ncbi.nlm.nih.gov/8512926" class="external text external">8512926</a>.</cite></span></li><li id="cite_note-Lobel_1996-5"> <span id="mw-reference-text-cite_note-Lobel_1996-5" class="mw-reference-text"><cite id="CITEREFLobellCrout1996" class="citation journal cs1">Lobell M, Crout DH (1996). "Pyruvate Decarboxylase: A Molecular Modeling Study of Pyruvate Decarboxylation and Acyloin Formation". <i>J. Am. Chem. Soc</i>. <b>118</b> (8): 1867–1873. doi:<a href="javascript:if(confirm('https://doi.org/10.1021%2Fja951830t  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://doi.org/10.1021%2Fja951830t'" tppabs="https://doi.org/10.1021%2Fja951830t" class="external text external">10.1021/ja951830t</a>.</cite></span></li><li id="cite_note-pmid9477950-6"> <span id="mw-reference-text-cite_note-pmid9477950-6" class="mw-reference-text"><cite id="CITEREFBaburinaDikdanGuoTous1998" class="citation journal cs1">Baburina I, Dikdan G, Guo F, Tous GI, Root B, Jordan F (February 1998). "Reactivity at the substrate activation site of yeast pyruvate decarboxylase: inhibition by distortion of domain interactions". <i>Biochemistry</i>. <b>37</b> (5): 1245–55. doi:<a href="javascript:if(confirm('https://doi.org/10.1021%2Fbi9709912  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://doi.org/10.1021%2Fbi9709912'" tppabs="https://doi.org/10.1021%2Fbi9709912" class="external text external">10.1021/bi9709912</a>. PMID<span>&nbsp;</span><a href="javascript:if(confirm('https://pubmed.ncbi.nlm.nih.gov/9477950  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://pubmed.ncbi.nlm.nih.gov/9477950'" tppabs="https://pubmed.ncbi.nlm.nih.gov/9477950" class="external text external">9477950</a>.</cite></span></li><li id="cite_note-7"> <span id="mw-reference-text-cite_note-7" class="mw-reference-text"><cite id="CITEREFH.2013" class="citation book cs1">H., Garrett, Reginald (2013). <i>Biochemistry</i>. Grisham, Charles M. (5th ed.). Belmont, CA: Brooks/Cole, Cengage Learning. ISBN<span>&nbsp;</span><bdi>9781133106296</bdi>. OCLC<span>&nbsp;</span><a href="javascript:if(confirm('https://www.worldcat.org/oclc/777722371  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://www.worldcat.org/oclc/777722371'" tppabs="https://www.worldcat.org/oclc/777722371" class="external text external">777722371</a>.</cite></span></li></ol></div>


    
</details><details data-level="2" open="">
    <summary class="section-heading"><h2 id="External_links">External links</h2></summary>
    
<ul><li><a href="javascript:if(confirm('https://meshb.nlm.nih.gov/record/ui?name=Pyruvate%20Decarboxylase  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://meshb.nlm.nih.gov/record/ui?name=Pyruvate%20Decarboxylase'" tppabs="https://meshb.nlm.nih.gov/record/ui?name=Pyruvate%20Decarboxylase" class="external text external">Pyruvate+Decarboxylase</a><span> at the US National Library of Medicine </span>Medical Subject Headings<span> (MeSH)</span></li></ul>

<div role="navigation" class="navbox" aria-labelledby="Carbon–carbon_lyases_(EC_4.1)" style="padding:3px"><table class="nowraplinks mw-collapsible autocollapse navbox-inner" style="border-spacing:0;background:transparent;color:inherit"><tbody><tr><th scope="col" class="navbox-title" colspan="2"><div id="Carbon–carbon_lyases_(EC_4.1)" style="font-size:114%;margin:0 4em">Carbon–carbon lyases (EC 4.1)</div></th></tr><tr><th scope="row" class="navbox-group" style="width:1%">4.1.1: Carboxy-lyases</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Acetoacetate decarboxylase</li>
<li>Adenosylmethionine decarboxylase</li>
<li>Arginine decarboxylase</li>
<li>Aromatic L-amino acid decarboxylase</li>
<li>Glutamate decarboxylase</li>
<li>Histidine decarboxylase</li>
<li>Lysine decarboxylase</li>
<li>Malonyl-CoA decarboxylase</li>
<li>Ornithine decarboxylase</li>
<li>Oxaloacetate decarboxylase</li>
<li>Phosphoenolpyruvate carboxykinase</li>
<li>Phosphoenolpyruvate carboxylase</li>
<li>Phosphoribosylaminoimidazole carboxylase</li>
<li>Pyrophosphomevalonate decarboxylase</li>
<li><a href="Pyruvate_decarboxylase.htm" tppabs="https://ptable.com/wiki/compounds/A/Pyruvate_decarboxylase" title="Pyruvate decarboxylase">Pyruvate decarboxylase</a></li>
<li>RuBisCO</li>
<li>Uridine monophosphate synthetase/Orotidine 5'-phosphate decarboxylase</li>
<li>Uroporphyrinogen III decarboxylase</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">4.1.2: <a href="Aldehyde.htm" tppabs="https://ptable.com/wiki/compounds/A/Aldehyde" title="Aldehyde">Aldehyde</a>-lyases</th><td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Fructose-bisphosphate aldolase
<ul><li>Aldolase A</li>
<li>Aldolase B</li>
<li>Aldolase C</li></ul></li>
<li>2-hydroxyphytanoyl-CoA lyase</li>
<li>Threonine aldolase</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">4.1.3: Oxo-acid-lyases</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Isocitrate lyase</li>
<li>3-hydroxy-3-methylglutaryl-CoA lyase</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">4.1.99: Other</th><td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Tryptophanase</li>
<li>Photolyase
<ul><li>CPD lyase</li>
<li>Spore photoproduct lyase</li></ul></li></ul>
</div></td></tr></tbody></table></div>
<div role="navigation" class="navbox" aria-labelledby="Enzymes" style="padding:3px"><table class="nowraplinks mw-collapsible autocollapse navbox-inner" style="border-spacing:0;background:transparent;color:inherit"><tbody><tr><th scope="col" class="navbox-title" colspan="2"><div id="Enzymes" style="font-size:114%;margin:0 4em">Enzymes</div></th></tr><tr><th scope="row" class="navbox-group" style="width:1%">Activity</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Active site</li>
<li>Binding site</li>
<li>Catalytic triad</li>
<li>Oxyanion hole</li>
<li>Enzyme promiscuity</li>
<li>Catalytically perfect enzyme</li>
<li>Coenzyme</li>
<li>Cofactor</li>
<li>Enzyme catalysis</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Regulation</th><td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Allosteric regulation</li>
<li>Cooperativity</li>
<li>Enzyme inhibitor</li>
<li>Enzyme activator</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Classification</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>EC number</li>
<li>Enzyme superfamily</li>
<li>Enzyme family</li>
<li>List of enzymes</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Kinetics</th><td class="navbox-list navbox-even hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li>Enzyme kinetics</li>
<li>Eadie–Hofstee diagram</li>
<li>Hanes–Woolf plot</li>
<li>Lineweaver–Burk plot</li>
<li>Michaelis–Menten kinetics</li></ul>
</div></td></tr><tr><th scope="row" class="navbox-group" style="width:1%">Types</th><td class="navbox-list navbox-odd hlist" style="text-align:left;border-left-width:2px;border-left-style:solid;width:100%;padding:0px"><div style="padding:0em 0.25em">
<ul><li><b>EC1 Oxidoreductases</b> (list)</li>
<li><b>EC2 Transferases</b> (list)</li>
<li><b>EC3 Hydrolases</b> (list)</li>
<li><b>EC4 Lyases</b> (list)</li>
<li><b>EC5 Isomerases</b> (list)</li>
<li><b>EC6 Ligases</b> (list)</li>
<li><b>EC7 Translocases</b> (list)</li></ul>
</div></td></tr></tbody></table></div>


<style data-mw-deduplicate="TemplateStyles:r951705291">.mw-parser-output cite.citation{font-style:inherit}.mw-parser-output .citation q{quotes:"\"""\"""'""'"}.mw-parser-output .id-lock-free a,.mw-parser-output .citation .cs1-lock-free a{background-image:url("../../../../upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png"/*tpa=https://upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/9px-Lock-green.svg.png*/);background-image:linear-gradient(transparent,transparent),url("../../../../upload.wikimedia.org/wikipedia/commons/6/65/Lock-green.svg"/*tpa=https://upload.wikimedia.org/wikipedia/commons/6/65/Lock-green.svg*/);background-repeat:no-repeat;background-size:9px;background-position:right .1em center}.mw-parser-output .id-lock-limited a,.mw-parser-output .id-lock-registration a,.mw-parser-output .citation .cs1-lock-limited a,.mw-parser-output .citation .cs1-lock-registration a{background-image:url("../../../../upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/9px-Lock-gray-alt-2.svg.png"/*tpa=https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/Lock-gray-alt-2.svg/9px-Lock-gray-alt-2.svg.png*/);background-image:linear-gradient(transparent,transparent),url("../../../../upload.wikimedia.org/wikipedia/commons/d/d6/Lock-gray-alt-2.svg"/*tpa=https://upload.wikimedia.org/wikipedia/commons/d/d6/Lock-gray-alt-2.svg*/);background-repeat:no-repeat;background-size:9px;background-position:right .1em center}.mw-parser-output .id-lock-subscription a,.mw-parser-output .citation .cs1-lock-subscription a{background-image:url("../../../../upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-2.svg/9px-Lock-red-alt-2.svg.png"/*tpa=https://upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Lock-red-alt-2.svg/9px-Lock-red-alt-2.svg.png*/);background-image:linear-gradient(transparent,transparent),url("../../../../upload.wikimedia.org/wikipedia/commons/a/aa/Lock-red-alt-2.svg"/*tpa=https://upload.wikimedia.org/wikipedia/commons/a/aa/Lock-red-alt-2.svg*/);background-repeat:no-repeat;background-size:9px;background-position:right .1em center}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration{color:#555}.mw-parser-output .cs1-subscription span,.mw-parser-output .cs1-registration span{border-bottom:1px dotted;cursor:help}.mw-parser-output .cs1-ws-icon a{background-image:url("../../../../upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/12px-Wikisource-logo.svg.png"/*tpa=https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/12px-Wikisource-logo.svg.png*/);background-image:linear-gradient(transparent,transparent),url("../../../../upload.wikimedia.org/wikipedia/commons/4/4c/Wikisource-logo.svg"/*tpa=https://upload.wikimedia.org/wikipedia/commons/4/4c/Wikisource-logo.svg*/);background-repeat:no-repeat;background-size:12px;background-position:right .1em center}.mw-parser-output code.cs1-code{color:inherit;background:inherit;border:inherit;padding:inherit}.mw-parser-output .cs1-hidden-error{display:none;font-size:100%}.mw-parser-output .cs1-visible-error{font-size:100%}.mw-parser-output .cs1-maint{display:none;color:#33aa33;margin-left:0.3em}.mw-parser-output .cs1-subscription,.mw-parser-output .cs1-registration,.mw-parser-output .cs1-format{font-size:95%}.mw-parser-output .cs1-kern-left,.mw-parser-output .cs1-kern-wl-left{padding-left:0.2em}.mw-parser-output .cs1-kern-right,.mw-parser-output .cs1-kern-wl-right{padding-right:0.2em}.mw-parser-output .citation .mw-selflink{font-weight:inherit}</style>
    
</details><!--htdig_noindex--><div><div style="clear:both; background-image:linear-gradient(180deg, #E8E8E8, white); border-top: dashed 2px #AAAAAA; padding: 0.5em 0.5em 0.5em 0.5em; margin-top: 1em; direction: ltr;">
    This article is issued from <a class="external text" title="Last edited on 2020-08-06" href="javascript:if(confirm('https://en.wikipedia.org/wiki/?title=Pyruvate_decarboxylase&oldid=971498582  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://en.wikipedia.org/wiki/?title=Pyruvate_decarboxylase&oldid=971498582'" tppabs="https://en.wikipedia.org/wiki/?title=Pyruvate_decarboxylase&oldid=971498582">Wikipedia</a>. The text is licensed under <a class="external text" href="javascript:if(confirm('https://creativecommons.org/licenses/by-sa/4.0/  \n\nThis file was not retrieved by Teleport VLX, because it is addressed on a domain or path outside the boundaries set for its Starting Address.  \n\nDo you want to open it from the server?'))window.location='https://creativecommons.org/licenses/by-sa/4.0/'" tppabs="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons - Attribution - Sharealike</a>. Additional terms may apply for the media files.
</div>
</div><!--/htdig_noindex--></div>
        </div>
      </div>
    </div>
  </div>
  <script src="../-/j/js_modules/jsConfigVars.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/jsConfigVars.js" class=""></script>
  <script src="../-/j/js_modules/startup.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/startup.js" class=""></script>
<script src="../-/j/js_modules/jquery.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/jquery.js" class=""></script>
<script src="../-/j/js_modules/mediawiki.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/mediawiki.js" class=""></script>
<script src="../-/j/js_modules/site.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/site.js" class=""></script>
<script src="../-/j/js_modules/ext.cite.ux-enhancements.js" tppabs="https://ptable.com/wiki/compounds/-/j/js_modules/ext.cite.ux-enhancements.js" class=""></script>



</body></html>